test/components/__snapshots__/userComments.test.js.snap

```diff
@@ -448,7 +399,9 @@ ShallowWrapper {
           <div
             className="c-header-divider"
           />,
-          <ArticleComment />,
+          <ArticleComment
+            postComment={undefined}
```

Comment:
I would update your test so that postComment comes through, rather than being undefined